### PR TITLE
CN-7208: screen names to constants  👀 👀 👀 👀 

### PR DESCRIPTION
**Pull request by srkela**
## Checklist (Mandatory):

- [ ] I have named the PR correctly (Check PR1)
- [ ] Check the commit’s or even all commits message styles matches our requested structure. (issue number, etc.) (Check CR10)
- [ ] I have tested new code locally (Check CR3)
- [ ] I have done a smoke test of the larger feature locally (Check CR3)
- [ ] I have added at least two reviewers (Check PR5)
- [ ] I have assigned the PR to myself (Check PR4)
- [ ] I have merged develop (parent) branch into my feature branch and resolved all conflicts (Check PR7)

## Checklist (Optional):

- [ ] I have added necessary documentation, including instructions how to test (Check CR8)
- [ ] I have added tests that prove my fix/implementation is effective or that my feature works
- [ ] I have written the code according to the adopted style guide? (Check CR7)

## Additional notes

> What does this implement/fix? Explain your changes. (optional)
> Are there any scripts that need running (eg. yarn install)?
> Are there any edge cases that need checking?
> Are there any specific test instructions?

[MVPWorkshop Pull Request & Code Review Guidelines](https://docs.google.com/document/d/1TcuU1JBhlGZLtpy0osSNinXQ2piP_yrDuRcXdqaOS_U/edit#heading=h.5b6kpdr4btq8)

### CODE REVIEW

**Review by fjovakaric — CHANGES_REQUESTED** [review]

File: app/components/atoms/InterestRateInfo/InterestRateInfo.js [hunk 1]
```diff
@@ -11,9 +11,8 @@ import Card from "../Card/Card";
 import CoinIcon from "../CoinIcon/CoinIcon";
 import cryptoUtil from "../../../utils/crypto-util";
 import RateInfoCard from "../../molecules/RateInfoCard/RateInfoCard";
-import { isUSCitizen } from "../../../utils/user-util/user-util";
-import { getColor } from "../../../utils/styles-util";
-import { COLOR_KEYS } from "../../../constants/COLORS";
+import { isUSCitizen } from "../../../utils/user-util";
```
Comment (fjovakaric) [inline app/components/atoms/InterestRateInfo/InterestRateInfo.js:14]: wrong path

File: app/components/molecules/CelSelect/CelSelect.js [hunk 2]
```diff
@@ -3,6 +3,7 @@ import { View, TouchableOpacity, Image, StyleSheet } from "react-native";
 import PropTypes from "prop-types";
 import { connect } from "react-redux";
 import { bindActionCreators } from "redux";
+import RNPickerSelect from "react-native-picker-select";
```
Comment (fjovakaric) [inline app/components/molecules/CelSelect/CelSelect.js:6]: My changes have been deleted :(

File: app/components/molecules/CoinGridCard/CoinGridCard.js [hunk 3]
```diff
@@ -16,7 +16,7 @@ import Counter from "../Counter/Counter";
 import animationsUtil from "../../../utils/animations-util";
 import ThemedImage from "../../atoms/ThemedImage/ThemedImage";
 import { COLOR_KEYS } from "../../../constants/COLORS";
-import { isUSCitizen } from "../../../utils/user-util/user-util";
+import { isUSCitizen } from "../../../utils/user-util";
```
Comment (fjovakaric) [inline app/components/molecules/CoinGridCard/CoinGridCard.js:19]: wrong path

File: app/components/molecules/CoinListCard/CoinListCard.js [hunk 4]
```diff
@@ -14,7 +14,7 @@ import { getColor, widthPercentageToDP } from "../../../utils/styles-util";
 import Counter from "../Counter/Counter";
 import animationsUtil from "../../../utils/animations-util";
 import { COLOR_KEYS } from "../../../constants/COLORS";
-import { isUSCitizen } from "../../../utils/user-util/user-util";
+import { isUSCitizen } from "../../../utils/user-util";
```
Comment (fjovakaric) [inline app/components/molecules/CoinListCard/CoinListCard.js:17]: wrong path

File: app/components/molecules/InterestCard/InterestCard.js [hunk 5]
```diff
@@ -10,12 +10,13 @@ import * as appActions from "../../../redux/actions";
 import { getColor, widthPercentageToDP } from "../../../utils/styles-util";
 import formatter from "../../../utils/formatter";
 import Spinner from "../../atoms/Spinner/Spinner";
-import { isUSCitizen } from "../../../utils/user-util/user-util";
+import { isUSCitizen } from "../../../utils/user-util";
```
Comment (fjovakaric) [inline app/components/molecules/InterestCard/InterestCard.js:13]: wrong path

File: app/components/molecules/KYCTrigger/KYCTrigger.js [hunk 6]
```diff
@@ -4,9 +4,10 @@ import PropTypes from "prop-types";
 import { KYC_STATUSES } from "../../../constants/DATA";
 import { MODALS } from "../../../constants/UI";
 import Banner from "../Banner/Banner";
-import { hasPassedKYC } from "../../../utils/user-util/user-util";
+import { hasPassedKYC } from "../../../utils/user-util";
```
Comment (fjovakaric) [inline app/components/molecules/KYCTrigger/KYCTrigger.js:7]: wrong path

File: app/components/molecules/PerCoinCelInterestCard/PerCoinCelInterestCard.js [hunk 7]
```diff
@@ -11,13 +11,14 @@ import Separator from "../../atoms/Separator/Separator";
 import CelButton from "../../atoms/CelButton/CelButton";
 import CelCheckbox from "../../atoms/CelCheckbox/CelCheckbox";
 import Icon from "../../atoms/Icon/Icon";
-import { isUSResident } from "../../../utils/user-util/user-util";
+import { isUSResident } from "../../../utils/user-util";
```
Comment (fjovakaric) [inline app/components/molecules/PerCoinCelInterestCard/PerCoinCelInterestCard.js:14]: wrong path

File: app/components/molecules/RateInfoCard/RateInfoCard.js [hunk 8]
```diff
@@ -10,10 +10,11 @@ import CelText from "../../atoms/CelText/CelText";
 import interestUtil from "../../../utils/interest-util";
 import formatter from "../../../utils/formatter";
 import CelButton from "../../atoms/CelButton/CelButton";
-import { isUSCitizen } from "../../../utils/user-util/user-util";
+import { isUSCitizen } from "../../../utils/user-util";
```
Comment (fjovakaric) [inline app/components/molecules/RateInfoCard/RateInfoCard.js:13]: wrong path

File: app/components/molecules/SocialSecurityNumber/SocialSecurityNumber.js [hunk 9]
```diff
@@ -9,7 +9,7 @@ import SocialSecurityNumberStyle from "./SocialSecurityNumber.styles";
 import CelText from "../../atoms/CelText/CelText";
 import CelInput from "../../atoms/CelInput/CelInputText";
 import CelButton from "../../atoms/CelButton/CelButton";
-import { isUSCitizen } from "../../../utils/user-util/user-util";
+import { isUSCitizen } from "../../../utils/user-util";
```
Comment (fjovakaric) [inline app/components/molecules/SocialSecurityNumber/SocialSecurityNumber.js:12]: wrong path

File: app/components/organisms/BannerCrossroad/BannerCrossroad.js [hunk 10]
```diff
@@ -4,7 +4,7 @@ import { bindActionCreators } from "redux";
 import moment from "moment";
 
 import * as appActions from "../../../redux/actions";
-import { hasPassedKYC } from "../../../utils/user-util/user-util";
+import { hasPassedKYC } from "../../../utils/user-util";
```
Comment (fjovakaric) [inline app/components/organisms/BannerCrossroad/BannerCrossroad.js:7]: wrong path

File: app/components/organisms/FabMenu/FabMenu.js [hunk 11]
```diff
@@ -17,12 +17,13 @@ import Fab from "../../molecules/Fab/Fab";
 import CircleButton from "../../atoms/CircleButton/CircleButton";
 import { THEMES } from "../../../constants/UI";
 import { KYC_STATUSES } from "../../../constants/DATA";
-import { hasPassedKYC, isKYCRejectedForever } from "../../../utils/user-util/user-util";
+import { hasPassedKYC, isKYCRejectedForever } from "../../../utils/user-util";
```
Comment (fjovakaric) [inline app/components/organisms/FabMenu/FabMenu.js:20]: wrong path

File: app/components/organisms/FabMenuAnimated/FabMenuAnimated.js [hunk 12]
```diff
@@ -18,7 +18,7 @@ import Fab from "../../molecules/Fab/Fab";
 import CircleButton from "../../atoms/CircleButton/CircleButton";
 import { THEMES } from "../../../constants/UI";
 import { KYC_STATUSES } from "../../../constants/DATA";
-import { hasPassedKYC, isKYCRejectedForever } from "../../../utils/user-util/user-util";
+import { hasPassedKYC, isKYCRejectedForever } from "../../../utils/user-util";
```
Comment (fjovakaric) [inline app/components/organisms/FabMenuAnimated/FabMenuAnimated.js:21]: wrong path

**Review by fjovakaric — CHANGES_REQUESTED** [review]

File: app/components/screens/ConfirmCamera/ConfirmCamera.js [hunk 13]
```diff
@@ -32,7 +33,7 @@ class ConfirmCamera extends Component {
   static navigationOptions = ({ navigation }) => {
     const documentPicture = navigation.getParam("documentPicture");
     const navigate = documentPicture
-      ? navigateTo("KYCVerifyIdentity")
+      ? navigateTo(SCREENS.KYC_VERIFICATION_IDENTITY)
```
Comment (fjovakaric) [inline app/components/screens/ConfirmCamera/ConfirmCamera.js:36]: should be KYC_VERIFY_IDENTITY

File: app/components/screens/InterestCalculatorScreen/InterestCalculatorScreen.js [hunk 14]
```diff
@@ -76,7 +77,9 @@ class InterestCalculatorScreen extends Component {
           button: (
             <CelButton
               margin="20 0 0 0"
-              onPress={() => actions.navigateTo("KYCVerifyID")}
+              onPress={() =>
+                actions.navigateTo(SCREENS.KYC_VERIFICATION_IDENTITY)
```
Comment (fjovakaric) [inline app/components/screens/InterestCalculatorScreen/InterestCalculatorScreen.js:81]: KYC_VERIFY_IDENTITY

File: app/components/screens/KYCAddressProof/KYCAddressProof.js [hunk 15]
```diff
@@ -34,7 +35,7 @@ class KYCAddressProof extends Component {
     customCenterComponent: { steps: 7, currentStep: 5, flowProgress: false },
     headerSameColor: true,
     customBack: () => {
-      store.dispatch(navigateTo("KYCVerifyIdentity"));
+      store.dispatch(navigateTo(SCREENS.KYC_VERIFICATION_IDENTITY));
```
Comment (fjovakaric) [inline app/components/screens/KYCAddressProof/KYCAddressProof.js:38]: KYC_VERIFY_IDENTITY

File: app/components/screens/KYCVerifyIdentity/KYCVerifyIdentity.js [hunk 16]
```diff
@@ -55,7 +56,7 @@ class KYCVerifyIdentity extends Component {
     const { actions, activeScreen, navigation } = this.props;
     if (
       activeScreen !== nextProps.activeScreen &&
-      nextProps.activeScreen === "KYCVerifyIdentity"
+      nextProps.activeScreen === SCREENS.KYC_VERIFICATION_IDENTITY
```
Comment (fjovakaric) [inline app/components/screens/KYCVerifyIdentity/KYCVerifyIdentity.js:59]: KYC_VERIFY_IDENTITY

File: app/constants/SCREENS.js [hunk 17]
```diff
@@ -0,0 +1,120 @@
+export const SCREENS = {
```
Comment (fjovakaric) [inline app/constants/SCREENS.js:1]: add comment with the link to the App screens spreadsheet

File: app/constants/SCREENS.js [hunk 18]
```diff
@@ -0,0 +1,120 @@
...
+  INTEREST_RATES: "InterestRates",
+  KYC_ADDRESS_INFO: "KYCAddressInfo",
+  KYC_ADDRESS_PROOF: "KYCAddressProof",
+  KYC_CHECK_PHOTOS: "KYCCheckPhotos", // note: do we use this?
+  KYC_CHECK_UTILITY_BILL: "KYCCheckUtilityBill", // note: do we use this?
+  KYC_FINAL_REJECTION: "KYCFinalRejection",
+  KYC_PRIME_TRUST_TOU: "KYCPrimeTrustToU",
+  KYC_PROFILE_DETAILS: "KYCProfileDetails",
+  KYC_TAXPAYER: "KYCTaxpayer",
+  KYC_VERIFICATION_IDENTITY: "KYCVerifyIdentity",
```
Comment (fjovakaric) [inline app/constants/SCREENS.js:57]: KYC_VERIFY_IDENTITY

File: app/constants/SCREENS.js [hunk 19]
```diff
@@ -0,0 +1,120 @@
...
+  SECURITY_OVERVIEW: "SecurityOverview",
+  SECURITY_SETTINGS: "SecuritySettings",
+  SELECT_COIN: "SelectCoin",
+  SELECT_COUNTRY: "SelectCountry",
+  SELECT_STATE: "SelectState",
+  SIMPLEX: "Simplex",
+  SIX_DIGIT_PIN_EXPLANATION: "SixDigitPinExplanation",
+  SPLASH_SCREEN: "SplashScreen",
+  STATIC_SCREEN: "StaticScreen", // NOTE: not used with navigateTo\
+  STORYBOOK: "storybook",
```
Comment (fjovakaric) [inline app/constants/SCREENS.js:92]: Storybook with `S`?

File: app/constants/SCREENS.js [hunk 20]
```diff
@@ -0,0 +1,120 @@
...
+  TRANSACTION_DETAILS_INTEREST: "TransactionDetailsInterest",
+  TRANSACTION_DETAILS_LOANS: "TransactionDetailsLoans",
+  TRANSACTION_DETAILS_REWARDS: "TransactionDetailsRewards",
+  TRANSACTION_DETAILS_WITHDRAW: "TransactionDetailsWithdraw",
+  TRANSACTION_INTERSECTION: "TransactionsIntersection",
+  TRANSACTION_ON_HOLD: "TransactionsOnHold", // NOTE: do we use this?
+  TWO_FA_AUTH_APP_CONFIRMATION_CODE: "TwoFaAuthAppConfirmationCode",
+  TWO_FACTOR_SETTINGS: "TwoFactorSettings",
+  VERIFY_PROFILE: "VerifyProfile",
+  WALLET: "Wallet", // NOTE: WTF AGAIN?
```
Comment (fjovakaric) [inline app/constants/SCREENS.js:107]: Remove screen if not used

File: app/utils/interest-util.js [hunk 21]
```diff
@@ -54,10 +54,9 @@ function getUserInterestForCoin(coinShort) {
     );
     specialRate = interestRates[coinShort].compound_cel_rate;
     if (interestRates[coinShort].threshold_on_first_n_coins) {
-      specialRate =
```
Comment (fjovakaric) [inline app/utils/interest-util.js:57]: should be this side

File: app/utils/interest-util.js [hunk 22]
```diff
@@ -90,18 +89,16 @@ function getLoyaltyRates(loyaltyInfo) {
 
   Object.keys(interestRates).forEach(coinShort => {
     const baseRate = interestUtil.getBaseCelRate(coinShort);
-    if (coinShort) {
```
Comment (fjovakaric) [inline app/utils/interest-util.js:93]: should be this side

File: app/utils/recaptcha-util.js [hunk 23]
```diff
@@ -7,7 +7,8 @@ import { getColor } from "./styles-util";
...
...DEBpnNzMDEBrHNzMDEBsgAAAAIAAAABAAAAAgAAAAEAAAACAAAAAQAAAAIAAAABAAAAAgAAAAEAAAACAAAAAQAAAAIAAAABAAAAAgAAAAEAAAACAAAAAQAAAAEAAgAAAAEACQAAAAEACAAAAAEABQAAAAEABAAAAAEAAwAAAAEABgAAAAEABwAAAAEACgAAAAEACgAAAAEACgAAAAEACgAAAAEACgAAAAEACgAAAAEACgAAAAEACgAAAAEACgAAAAEACwAAAAEACwAAAAEACwAAAAEACwAAAAEACwAAAAEACwAAAAEACwAAAAEACwAAAAEACwANABwAJAAsADYAPgBGAE4AVgBeAGYAbgB2AH4AAQAAAAEBZgADAAAAAQG6AAYAAAACAFoAbAABAAAAAQB2AAEAAAABAHQAAQAAAAEAcgABAAAAAQBwAAEAAAABAG4AAQAAAAEAbAABAAAAAQBqAAEAAAABAGgAAQAAAAEAtgABAAAAAQFwAAMAAAABAW4AAQF0AAEAAAAMAAMAAAABAVwAAgF+AWIAAQAAAAwAAQF6AAEAAQF0AAEAAQE8AAUAAQE2AAUAAQEwAAUAAQEqAAUAAQEkAAUAAgFcACgA6gDrAOwA7QDuAO8A8ADxAPIA8wD0APUA9gD3APgA+QD6APsA/AD9AP4A/wEAAQEBAgEDAQQBBQEGAQcBCAEJAQoBCwEMAQ0BDgEPARABEQACAQYAKADqAOsA7ADtAO4A7wDwAPEA8gDzAPQA9QD2APcA+AD5APoA+wD8AP0A/gD/AQABAQECAQMBBAEFAQYBBwEIAQkBCgELAQwBDQEOAQ8BEAERAAIA0gArAFQAWQDqAOsA7ADtAO4A7wDwAPEA8gDzAPQA9QD2APcA+AD5APoA+wD8AP0A/gD/AQABAQECAQQBBQEGAQcBCAEJAQoBCwEMAQ0BDgEPARABEQDIAM4AAQAWAAEACAADAJwAoAEDAAEABgABAAEAAQCbAAEADAFwAXEBcgF0AXYBdwF4AXkBegF7AX0BggABAAYBcwF8AX4BfwGAAYEAAQAEAFMAWADHAM0AAgAFAHUAfwAAAIoAkwALAJUAmAAVAJsApAAZAKcAqwAjAAIACQBTAFMAAABYAFgAAQB1AH8AAgCKAJMADQCVAJgAFwCcAKQAGwCnAKsAJADHAMcAKQDNAM0AKg=="
```
Comment (fjovakaric) [inline app/utils/recaptcha-util.js:10]: should be this side

File: app/utils/transaction-map-util.js [hunk 24]
```diff
@@ -320,17 +320,17 @@ function getTransactionProps(transaction) {
       };
     case TRANSACTION_TYPES.INTEREST:
       return {
-        title: coin => `${coin} Reward`,
```
Comment (fjovakaric) [inline app/utils/transaction-map-util.js:323]: should be this side

File: app/utils/user-util/user-util.test.js [hunk 25]
```diff
@@ -1,4 +1,4 @@
-import _ from "lodash"
```
Comment (fjovakaric) [inline app/utils/user-util/user-util.test.js:1]: why change this file

**Review by fjovakaric — CHANGES_REQUESTED** [review]

File: app/utils/user-util/user-util.test.js (hunk 25 above)
Comment (srkela) [reply app/utils/user-util/user-util.test.js:1]: prettier reformatted file when pre-commit

Comment (srkela) [reply app/utils/user-util/user-util.test.js:1]: prettier reformatted file when pre-commit

**Review by fjovakaric — APPROVED** [review]

**Review by CrnogoracIvan — CHANGES_REQUESTED** [review]
Check storybook on two places and KYC_IDENTITY or KYC_ID. Check comments.

File: app/components/screens/ActionsByUser/ActionsByUser.stories.js [hunk 26]
```diff
@@ -27,7 +27,7 @@ const regular = () => {
 
   return (
     <ScreenStoryWrapper
-      screenName="VerifyProfile"
+      screenName="ActionsByUser"
```
[the remark this answers is not in the document]
Comment (srkela) [reply app/components/screens/ActionsByUser/ActionsByUser.stories.js:30]: Stories was not covered. You wanted do check another ~100 files? 😂

File: app/components/screens/CellphoneEnter/CellphoneEnter.stories.js [hunk 27]
```diff
@@ -14,7 +14,7 @@ const initialState = {
 const regular = () => {
   return (
     <ScreenStoryWrapper
-      screenName="CellphoneEnter"
+      screenName={"CellphoneEnter"}
```
[the remark this answers is not in the document]
Comment (srkela) [reply app/components/screens/CellphoneEnter/CellphoneEnter.stories.js:17]: Same here

File: app/components/screens/InterestCalculatorScreen/InterestCalculatorScreen.js [hunk 28]
```diff
@@ -76,7 +77,7 @@ class InterestCalculatorScreen extends Component {
           button: (
             <CelButton
               margin="20 0 0 0"
-              onPress={() => actions.navigateTo("KYCVerifyID")}
+              onPress={() => actions.navigateTo(SCREENS.KYC_VERIFY_IDENTITY)}
```
[the remark this answers is not in the document]
Comment (srkela) [reply app/components/screens/InterestCalculatorScreen/InterestCalculatorScreen.js:80]: `KYCVerifyID.js` removed, new screen is  `KYCVerifyIdentity.js`

**Review by CrnogoracIvan — APPROVED** [review]
